lib/web/mage/tabs.js

```diff
@@ -71,6 +71,9 @@ define([
                 anchorId = anchor.replace('#', '');
 
             if (anchor && isValid) {
+				if(anchorId == 'review-form'){
```

Comment:
Hi @prakash2jcommerce. Thanks for collaboration.
First of all, according to our [standarts](https://devdocs.magento.com/guides/v2.2/coding-standards/technical-guidelines.html) we should use `===` instead of `==`.

Also, for support flexibility, we should avoid hardcode tab id (`review-form`)